core/templates/base-components/base-content.component.css

```diff
@@ -41,6 +41,11 @@
     display: none;
   }
 }
+@media (max-width: 992px) {
+  .navbar-collapse {
+    display: flex !important; 
```

Comment:
Can we avoid the use of `!important` here? (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css))

Reply:
Yes, my apologies for that.